src/app/add-item/add-item.component.html

```diff
@@ -1,5 +1,6 @@
 <div class="container dialogModal">
-    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <h2 *ngIf="createDialog" class="dialogTitle" mat-dialog-title>{{ dialogTitleForAddingItems }}</h2>
```

Comment:
**SUGGESTION:** We could unify that into single html and single title value and place the if logic in the TypeScript code.